init/postgres/test_sample.sql

```diff
@@ -187,6 +187,7 @@ CREATE TABLE test_sample.employee (
   last_name VARCHAR (255) NOT NULL,
   employment_date timestamp with time zone,
   manager_id INT,
+  pto_accural interval,
   FOREIGN KEY (manager_id)
       REFERENCES test_sample.employee (employee_id)
       ON DELETE CASCADE
```

Comment:
Add some random intervals values bellow.

Reply:
Also don't forget to add the same for cockroachdb, since postgres tests are run on both databases.

Reply:
Done